# German translations for keystone.
# Copyright (C) 2013 ORGANIZATION
# This file is distributed under the same license as the keystone project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version:  Keystone\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2013-05-22 17:05+0000\n"
"PO-Revision-Date: 2013-05-22 03:45+0000\n"
"Last-Translator: daisy.ycguo <daisy.ycguo@gmail.com>\n"
"Language-Team: German "
"(http://www.transifex.com/projects/p/openstack/language/de/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"

#: keystone/clean.py:23
#, python-format
msgid "%s cannot be empty."
msgstr "%s darf nicht leer sein."

#: keystone/clean.py:25
#, python-format
msgid "%(property_name)s cannot be less than %(min_length)s characters."
msgstr "%(property_name)s darf nicht kleiner als %(min_length)s Zeichen sein."

#: keystone/clean.py:29
#, python-format
msgid "%(property_name)s should not be greater than %(max_length)s characters."
msgstr "%(property_name)s sollte nicht größer als %(max_length)s Zeichen sein."

#: keystone/clean.py:36
#, python-format
msgid "%(property_name)s is not a %(display_expected_type)s"
msgstr ""

#: keystone/test.py:115
#, python-format
msgid "Failed to checkout %s"
msgstr "%s wurde nicht ausgecheckt"

#: keystone/auth/controllers.py:72
#, python-format
msgid "Project is disabled: %s"
msgstr "Projekt ist inaktiviert: %s"

#: keystone/auth/controllers.py:78 keystone/auth/plugins/password.py:40
#, python-format
msgid "Domain is disabled: %s"
msgstr "Domäne ist inaktiviert: %s"

#: keystone/auth/controllers.py:84 keystone/auth/plugins/password.py:46
#, python-format
msgid "User is disabled: %s"
msgstr "Benutzer ist inaktiviert: %s"

#: keystone/auth/controllers.py:265
msgid "Scoping to both domain and project is not allowed"
msgstr "Scoping sowohl auf 'domain' als auch auf 'project' ist nicht zulässig"

#: keystone/auth/controllers.py:268
msgid "Scoping to both domain and trust is not allowed"
msgstr "Scoping sowohl auf 'domain' als auch auf 'trust' ist nicht zulässig"

#: keystone/auth/controllers.py:271
msgid "Scoping to both project and trust is not allowed"
msgstr "Scoping sowohl auf 'project' als auch auf 'trust' ist nicht zulässig"

#: keystone/auth/controllers.py:333
#, python-format
msgid "Unable to lookup user %s"
msgstr "Suche nach Benutzer %s nicht möglich"

#: keystone/auth/controllers.py:363
msgid "User not found"
msgstr "Benutzer nicht gefunden"

#: keystone/auth/token_factory.py:81
msgid "User have no access to project"
msgstr "Benutzer hat keinen Zugriff auf Projekt"

#: keystone/auth/token_factory.py:96
msgid "User have no access to domain"
msgstr "Benutzer hat keinen Zugriff auf Domäne"

#: keystone/auth/token_factory.py:314 keystone/token/controllers.py:121
msgid "Unable to sign token."
msgstr "Token kann nicht unterzeichnet werden."

#: keystone/auth/token_factory.py:317 keystone/token/controllers.py:124
#, python-format
msgid "Invalid value for token_format: %s.  Allowed values are PKI or UUID."
msgstr ""
"Ungültiger Wert für 'token_format': %s.  Zulässige Werte sind PKI oder "
"UUID."

#: keystone/catalog/core.py:38
#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr "Fehlerhafter Endpunkt %(url)s - unbekannter Schlüssel %(keyerror)s"

#: keystone/catalog/core.py:43
#, python-format
msgid ""
"Malformed endpoint %(url)s - unknown key %(keyerror)s(are you missing "
"brackets ?)"
msgstr ""
"Fehlerhafter Endpunkt %(url)s - unbekannter Schlüssel %(keyerror)s "
"(fehlende Klammer?)"

#: keystone/catalog/core.py:49
#, python-format
msgid ""
"Malformed endpoint %s - incomplete format                   (are you "
"missing a type notifier ?)"
msgstr ""
"Fehlerhafter Endpunkt %s - unvollständiges Format                   "
"(fehlender Typenhinweis?)"

#: keystone/catalog/backends/templated.py:109
#, python-format
msgid "Unable to open template file %s"
msgstr "Vorlagendatei %s kann nicht geöffnet werden"

#: keystone/common/bufferedhttp.py:102
#, python-format
msgid "HTTP PERF: %(time).5f seconds to %(method)s %(host)s:%(port)s %(path)s)"
msgstr "HTTP PERF: %(time).5f Sekunden für %(method)s %(host)s:%(port)s %(path)s)"

#: keystone/common/cms.py:44
#, python-format
msgid "Verify error: %s"
msgstr "Fehler überprüfen: %s"

#: keystone/common/cms.py:135
msgid ""
"Signing error: Unable to load certificate - ensure you've configured PKI "
"with 'keystone-manage pki_setup'"
msgstr ""
"Unterzeichnungsfehler: Zertifikat kann nicht geladen werden - stellen Sie"
" sicher, dass Sie PKI mit 'keystone-manage pki_setup' konfiguriert haben"

#: keystone/common/cms.py:139
#, python-format
msgid "Signing error: %s"
msgstr "Unterzeichnungsfehler: %s"

#: keystone/common/config.py:93
#, python-format
msgid "Unable to locate specified logging config file: %s"
msgstr "Angegebene Protokollkonfigurationsdatei kann nicht gefunden werden: %s"

#: keystone/common/config.py:111
msgid "Invalid syslog facility"
msgstr "Ungültige 'syslog'-Funktion"

#: keystone/common/controller.py:19
#, python-format
msgid "RBAC: Authorizing %s(%s)"
msgstr "Rollenbasierte Zugriffssteuerung: Autorisierung von %s(%s)"

#: keystone/common/controller.py:27
msgid "RBAC: Invalid token"
msgstr "Rollenbasierte Zugriffssteuerung: Ungültiges Token"

#: keystone/common/controller.py:37 keystone/common/controller.py:58
msgid "RBAC: Invalid user"
msgstr "Rollenbasierte Zugriffssteuerung: Ungültiger Benutzer"

#: keystone/common/controller.py:43
msgid "RBAC: Proceeding without project"
msgstr "Rollenbasierte Zugriffssteuerung: Fortfahren ohne Projekt"

#: keystone/common/controller.py:63
msgid "RBAC: Proceeding without tenant"
msgstr "Rollenbasierte Zugriffssteuerung: Fortsetzung ohne Nutzer"

#: keystone/common/controller.py:93 keystone/common/controller.py:145
msgid "RBAC: Bypassing authorization"
msgstr "Rollenbasierte Zugriffssteuerung: Umgehen von Autorisierung"

#: keystone/common/controller.py:102 keystone/common/controller.py:143
msgid "RBAC: Authorization granted"
msgstr "Rollenbasierte Zugriffssteuerung: Autorisierung erteilt"

#: keystone/common/controller.py:132
#, python-format
msgid "RBAC: Adding query filter params (%s)"
msgstr ""
"Rollenbasierte Zugriffssteuerung: Hinzufügen von Abfragefilterparametern "
"(%s)"

#: keystone/common/controller.py:332
msgid "Invalid token in normalize_domain_id"
msgstr "Ungültiges Token in 'normalize_domain_id'"

#: keystone/common/wsgi.py:106
#, python-format
msgid "Starting %(arg0)s on %(host)s:%(port)s"
msgstr "Starten von %(arg0)s auf %(host)s:%(port)s"

#: keystone/common/wsgi.py:166
msgid "Server error"
msgstr "Serverfehler"

#: keystone/common/wsgi.py:245
#, python-format
msgid "arg_dict: %s"
msgstr "arg_dict: %s"

#: keystone/common/wsgi.py:267
#, python-format
msgid "Authorization failed. %s from %s"
msgstr "Autorisierung fehlgeschlagen. %s von %s"

#: keystone/common/wsgi.py:510
msgid "The resource could not be found."
msgstr "Die Ressource konnte nicht gefunden werden."

#: keystone/common/ldap/core.py:79
#, python-format
msgid "Invalid LDAP deref option: %s. Choose one of: "
msgstr "Ungültige LDAP-deref-Option: %s. Wählen Sie aus: "

#: keystone/common/ldap/core.py:87
#, python-format
msgid "Invalid LDAP tls certs option: %s. Choose one of: "
msgstr ""

#: keystone/common/ldap/core.py:96
#, python-format
msgid "Invalid LDAP scope: %s. Choose one of: "
msgstr "Ungültiger LDAP-Umfang: %s. Wählen Sie aus: "

#: keystone/common/ldap/core.py:185
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%s\". Format must be "
"<ldap_attribute>:<keystone_attribute>"
msgstr ""

#: keystone/common/ldap/core.py:190
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%(item)s\". Value "
"\"%(attr_map)s\" must use one of %(keys)s."
msgstr ""

#: keystone/common/ldap/core.py:274 keystone/identity/backends/kvs.py:603
#: keystone/identity/backends/kvs.py:631
#, python-format
msgid "Duplicate name, %s."
msgstr "Doppelter Name, %s."

#: keystone/common/ldap/core.py:284 keystone/identity/backends/kvs.py:596
#, python-format
msgid "Duplicate ID, %s."
msgstr "Doppelte ID, %s."

#: keystone/common/ldap/core.py:289
#, python-format
msgid "LDAP %s create"
msgstr "LDAP %s erstellen"

#: keystone/common/ldap/core.py:367
#, python-format
msgid "LDAP %s update"
msgstr "LDAP %s aktualisieren"

#: keystone/common/ldap/core.py:400
#, python-format
msgid "LDAP %s delete"
msgstr "LDAP %s löschen"

#: keystone/common/ldap/core.py:425
#, python-format
msgid "LDAP init: url=%s"
msgstr "LDAP starten: url=%s"

#: keystone/common/ldap/core.py:426
#, python-format
msgid ""
"LDAP init: use_tls=%(use_tls)s\n"
"tls_cacertfile=%(tls_cacertfile)s\n"
"tls_cacertdir=%(tls_cacertdir)s\n"
"tls_req_cert=%(tls_req_cert)s\n"
"tls_avail=%(tls_avail)s\n"
msgstr ""

#: keystone/common/ldap/core.py:445
msgid "Invalid TLS / LDAPS combination"
msgstr ""

#: keystone/common/ldap/core.py:449
#, python-format
msgid "Invalid LDAP TLS_AVAIL option: %s. TLSnot available"
msgstr ""

#: keystone/common/ldap/core.py:459
#, python-format
msgid "tls_cacertfile %s not found or is not a file"
msgstr ""

#: keystone/common/ldap/core.py:471
#, python-format
msgid "tls_cacertdir %s not found or is not a directory"
msgstr ""

#: keystone/common/ldap/core.py:478
#, python-format
msgid "LDAP TLS: invalid TLS_REQUIRE_CERT Option=%s"
msgstr ""

#: keystone/common/ldap/core.py:492
#, python-format
msgid "LDAP bind: dn=%s"
msgstr "LDAP binden: dn=%s"

#: keystone/common/ldap/core.py:503
#, python-format
msgid "LDAP add: dn=%s, attrs=%s"
msgstr "LDAP hinzufügen: dn=%s, attrs=%s"

#: keystone/common/ldap/core.py:508
#, python-format
msgid "LDAP search: dn=%s, scope=%s, query=%s, attrs=%s"
msgstr "LDAP-Suche: dn=%s, scope=%s, query=%s, attrs=%s"

#: keystone/common/ldap/core.py:559
msgid ""
"LDAP Server does not support paging. Disable paging in keystone.conf to "
"avoid this message."
msgstr ""
"LDAP-Server unterstützt Paging nicht. Inaktivieren Sie das Paging in "
"keystone.conf, um diese Nachricht zu verhindern."

#: keystone/common/ldap/core.py:576
#, python-format
msgid "LDAP modify: dn=%s, modlist=%s"
msgstr "LDAP ändern: dn=%s, modlist=%s"

#: keystone/common/ldap/core.py:581
#, python-format
msgid "LDAP delete: dn=%s"
msgstr "LDAP löschen: dn=%s"

#: keystone/common/ldap/core.py:585
#, python-format
msgid "LDAP delete_ext: dn=%s, serverctrls=%s"
msgstr "LDAP-'delete_ext': dn=%s, serverctrls=%s"

#: keystone/common/ldap/fakeldap.py:146
#, python-format
msgid "FakeLdap initialize url=%s"
msgstr "FakeLdap starten, url=%s"

#: keystone/common/ldap/fakeldap.py:156
#, python-format
msgid "FakeLdap bind dn=%s"
msgstr "FakeLdap binden, dn=%s"

#: keystone/common/ldap/fakeldap.py:163
#, python-format
msgid "FakeLdap bind fail: dn=%s not found"
msgstr "FakeLdap binden fehlgeschlagen: dn=%s nicht gefunden"

#: keystone/common/ldap/fakeldap.py:170
#, python-format
msgid "FakeLdap bind fail: password for dn=%s not found"
msgstr "FakeLdap binden fehlgeschlagen: Kennwort für dn=%s nicht gefunden"

#: keystone/common/ldap/fakeldap.py:175
#, python-format
msgid "FakeLdap bind fail: password for dn=%s does not match"
msgstr "FakeLdap binden fehlgeschlagen: Kennwort für dn=%s stimmt nicht überein"

#: keystone/common/ldap/fakeldap.py:190
#, python-format
msgid "FakeLdap add item: dn=%s, attrs=%s"
msgstr "FakeLdap-Element hinzufügen: dn=%s, attrs=%s"

#: keystone/common/ldap/fakeldap.py:192
#, python-format
msgid "FakeLdap add item failed: dn=%s is already in store."
msgstr "FakeLdap-Element hinzufügen fehlgeschlagen: dn=%s ist bereits im Speicher."

#: keystone/common/ldap/fakeldap.py:206 keystone/common/ldap/fakeldap.py:220
#, python-format
msgid "FakeLdap delete item: dn=%s"
msgstr "FakeLdap-Element löschen: dn=%s"

#: keystone/common/ldap/fakeldap.py:210 keystone/common/ldap/fakeldap.py:224
#, python-format
msgid "FakeLdap delete item failed: dn=%s not found."
msgstr "FakeLdap-Element löschen fehlgeschlagen: dn=%s nicht gefunden."

#: keystone/common/ldap/fakeldap.py:239
#, python-format
msgid "FakeLdap modify item: dn=%s attrs=%s"
msgstr "FakeLdap-Element ändern: dn=%s attrs=%s"

#: keystone/common/ldap/fakeldap.py:243
#, python-format
msgid "FakeLdap modify item failed: dn=%s not found."
msgstr "FakeLdap-Element ändern fehlgeschlagen: dn=%s nicht gefunden."

#: keystone/common/ldap/fakeldap.py:260
#, python-format
msgid "FakeLdap modify item failed: item has no attribute \"%s\" to delete"
msgstr ""
"FakeLdap-Element ändern fehlgeschlagen: Element hat kein zu löschendes "
"Attribut \"%s\""

#: keystone/common/ldap/fakeldap.py:271
#, python-format
msgid ""
"FakeLdap modify item failed: item has no attribute \"%s\" with value "
"\"%s\" to delete"
msgstr ""
"FakeLdap-Element ändern fehlgeschlagen: Element hat kein zu löschendes "
"Attribut \"%s\" mit Wert \"%s\""

#: keystone/common/ldap/fakeldap.py:276
#, python-format
msgid "FakeLdap modify item failed: unknown command %s"
msgstr "FakeLdap-Element ändern fehlgeschlagen: unbekannter Befehl %s"

#: keystone/common/ldap/fakeldap.py:278
#, python-format
msgid "modify_s action %s not implemented"
msgstr "'modify_s'-Aktion %s nicht implementiert"

#: keystone/common/ldap/fakeldap.py:296
#, python-format
msgid "FakeLdap search at dn=%s scope=%s query=%s"
msgstr "FakeLdap-Suche unter dn=%s scope=%s query=%s"

#: keystone/common/ldap/fakeldap.py:302
msgid "FakeLdap search fail: dn not found for SCOPE_BASE"
msgstr "FakeLdap-Suche fehlgeschlagen: dn für 'SCOPE_BASE' nicht gefunden"

#: keystone/common/ldap/fakeldap.py:316
#, python-format
msgid "Search scope %s not implemented."
msgstr "Suchbereich %s nicht implementiert."

#: keystone/common/sql/core.py:207
#, python-format
msgid "Got mysql server has gone away: %s"
msgstr ""
"Es wurde festgestellt, dass keine Verbindung zum mysql-Server mehr "
"vorhanden ist: %s"

#: keystone/common/sql/legacy.py:180
#, python-format
msgid "Cannot migrate EC2 credential: %s"
msgstr "EC2-Berechtigungsnachweis kann nicht migriert werden: %s"

#: keystone/common/sql/migration.py:47
msgid "version should be an integer"
msgstr "Version sollte eine Ganzzahl sein"

#: keystone/common/sql/nova.py:62
#, python-format
msgid "Create tenant %s"
msgstr "Nutzer %s erstellen"

#: keystone/common/sql/nova.py:79
#, python-format
msgid "Create user %s"
msgstr "Benutzer %s erstellen"

#: keystone/common/sql/nova.py:88
#, python-format
msgid "Add user %s to tenant %s"
msgstr "Benutzer %s zu Nutzer %s hinzufügen"

#: keystone/common/sql/nova.py:96
#, python-format
msgid "Ignoring existing role %s"
msgstr "Vorhandene Rolle %s ignorieren"

#: keystone/common/sql/nova.py:103
#, python-format
msgid "Create role %s"
msgstr "Rolle %s erstellen"

#: keystone/common/sql/nova.py:113
#, python-format
msgid "Assign role %s to user %s on tenant %s"
msgstr "Rolle %s Benutzer %s auf Nutzer %s zuweisen"

#: keystone/common/sql/nova.py:128
#, python-format
msgid "Creating ec2 cred for user %s and tenant %s"
msgstr "Erstellen von EC2-Berechtigungsnachweis für Benutzer %s und Nutzer %s"

#: keystone/identity/backends/kvs.py:257 keystone/identity/backends/kvs.py:266
msgid "User not found in group"
msgstr "Benutzer nicht in Gruppe gefunden"

#: keystone/identity/backends/sql.py:424
#, python-format
msgid "Cannot remove role that has not been granted, %s"
msgstr "Nicht gewährte Rolle kann nicht entfernt werden, %s"

#: keystone/identity/backends/ldap/core.py:95
#, python-format
msgid "Expected dict or list: %s"
msgstr ""

#: keystone/identity/backends/ldap/core.py:692
#, python-format
msgid "Role %s not found"
msgstr "Rolle %s nicht gefunden"

#: keystone/identity/backends/ldap/core.py:912
msgid "Changing Name not supported by LDAP"
msgstr "Änderung von Namen wird von LDAP nicht unterstützt"

#: keystone/identity/backends/ldap/core.py:926
#, python-format
msgid "User %s is already a member of group %s"
msgstr "Benutzer %s ist bereits Mitglied der Gruppe %s."

#: keystone/identity/backends/ldap/core.py:968
#, python-format
msgid ""
"Group member '%(user_dn)s' not found in '%(group_dn)s'. The user should "
"be removed from the group. The user will be ignored."
msgstr ""

#: keystone/openstack/common/policy.py:394
#, python-format
msgid "Failed to understand rule %(rule)s"
msgstr "Regel '%(rule)s' konnte nicht verstanden werden"

#: keystone/openstack/common/policy.py:404
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Kein Handler für Übereinstimmungen des Typs %s"

#: keystone/openstack/common/policy.py:679
#, python-format
msgid "Failed to understand rule %(rule)r"
msgstr "Regel '%(rule)r' konnte nicht verstanden werden"

#: keystone/policy/backends/rules.py:93
#, python-format
msgid "enforce %s: %s"
msgstr "%s erzwingen: %s"

#: keystone/token/controllers.py:466 keystone/token/controllers.py:469
msgid "Token does not belong to specified tenant."
msgstr "Token gehört nicht zu angegebenem Nutzer."

#: keystone/token/controllers.py:476
msgid "Non-default domain is not supported"
msgstr "Nicht-Standard-Domäne wird nicht unterstützt"

#: keystone/token/controllers.py:484
msgid "Domain scoped token is not supported"
msgstr "Bereichsorientiertes Token der Domäne wird nicht unterstützt"

#: keystone/token/backends/memcache.py:83
msgid "Unable to add token user list."
msgstr "Token kann nicht zu Benutzerliste hinzugefügt werden."

#: keystone/token/backends/memcache.py:93
msgid "Unable to add token to revocation list."
msgstr "Token kann nicht zu Widerrufsliste hinzugefügt werden."

